api/app/Http/Middleware/ProtectedRequest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+
+/**
+ * Notices if the request is a secure one that would have been
+ * protected by the VPN.
+ */
+class ProtectedRequest
+{
+    /**
+     * @return mixed Any kind of response
+     */
+    public function handle(Request $request, Closure $next)
+    {
+        $path = $request->path();
+        if (str_starts_with($path, 'admin')) {
```

Comment:
Maybe instead use `$request->is('admin/*')`. See https://laravel.com/docs/11.x/requests#inspecting-the-request-path. I'm just thinking about how our files like `adminPage.js` got flagged by the firewall 😅

Reply:
Oh, very good catch!  That's a nice little matcher function.